testing/apps/productpage/Dockerfile.sidecar

```diff
@@ -15,8 +15,8 @@
 FROM amalgam8/a8-sidecar:latest
 
 WORKDIR /opt/microservices
+COPY config.yaml /opt/microservices/
 COPY productpage /opt/microservices/
-COPY config.yaml /opt/microservices/config.yaml
-EXPOSE 9080
+EXPOSE 9080 12345
```

Comment:
12345 is a hack to test that the envoy proxy inside productpage can reach the tcphelloworld app in tests.  tcp listeners need to be configured via a config.yaml in sidecar.  We didn't want to modify the gateway image, so we modified productpage since it already had a config yaml to accomplish the same thing